---
description: LOC_Os02g50240 ; Os02g0735200 ; Oryza sativa
meta_title:
title: OsGS,OsGS1,GS1,OsGS1;1,OsGLN1;1
categories: [Oryza sativa]
tags: [stem, seed, leaf, nitrogen, drought, root, grain, growth, spikelet, sheath, chloroplast, grain filling, drought tolerance, cadmium, oxidative, seedling, salinity, tolerance, abiotic stress, stress, Al tolerance, biotic stress, salinity stress, transcription factor, development, photosynthesis, homeostasis, chloroplast development, sugar, amylose content]
---

## Gene Details:
- **MSU gene ID:** [LOC_Os02g50240](http://rice.uga.edu/cgi-bin/ORF_infopage.cgi?orf=LOC_Os02g50240)  
- **RAPdb gene ID:** [Os02g0735200](https://rapdb.dna.affrc.go.jp/locus/?name=Os02g0735200)  
- **Gene Symbol:** OsGS,OsGS1,GS1,OsGS1;1,OsGLN1;1
- **Gene Name:**
- **Genome:**  [MSU7](http://rice.uga.edu/)&nbsp;,&nbsp;[IRGSP-1.0](https://rapdb.dna.affrc.go.jp/download/irgsp1.html)
- **Species:** *Oryza sativa*

## Functional Descriptions:
   - Following WD, **OsGS1;1** mRNA level in stem and root tissues declined in IR-64 and enhanced in Khitish.
   - In both the genotypes, both increase and decline in seed protein contents matched with the expressions levels of OsAMT1;1, **OsGS1;1** and OsGS1;2 in the flag leaves during grain filling stage indicating that high nitrogen nutrition in KN3119 probably causes the repression of these genes which might be important during grain filling.
   - Overaccumulation of free ammonium in the leaf sheath and roots of the mutant indicated the importance of **OsGS1;1** for ammonium assimilation in both organs.
   - **OsGS1;1** and OsGS2 were differentially regulated in IR-64 (drought-sensitive) and Khitish (drought-tolerant) cultivars of rice.
   - Results suggest that OsGS2 and **OsGS1;1** expression may contribute to drought tolerance of Khitish cultivar under WD conditions.
   - A steady **OsGS1**;2 expression patterns were noted in leaf, stem and root of both the cultivars.
   - Severe reduction in growth rate and grain filling of rice mutants lacking **OsGS1;1**, a cytosolic glutamine synthetase1;1.
   - **OsGS1;1** was expressed in all organs tested with higher expression in leaf blades, while OsGS1;2, and OsGS1;3 were expressed mainly in roots and spikelets, respectively.
   - Of the three genes encoding cytosolic GS in rice, **OsGS1;1** is critical for normal growth and grain filling.
   - These results demonstrated a crucial function of **OsGS1;1** in coordinating the global metabolic network in rice plants grown using ammonium as the nitrogen source.
   - **GS1** transcript and polypeptide accumulated in leaf during WD, however, **GS1** activity was maintained at a constant level.
   - Among the three isoenzymes of the cytosolic GS1 gene family in rice, **OsGLN1;1** and OsGLN1;2 were abundantly expressed in roots.
   - The **OsGLN1;1** and OsGLN1;2 transcripts showed reciprocal responses to ammonium supply in the surface cell layers of roots.
   - The high-capacity Gln synthetic activities of **OsGLN1;1** and OsGLN1;2 facilitate active ammonium assimilation in specific cell types in rice roots.
   - Low-affinity forms of **GS1** comparable to those in Arabidopsis (GLN1;2 and GLN1;3) were absent in rice roots.
   - Rice plants possess one gene (OsGS2) for chloroplastic GS2 and three genes (**OsGS1;1**, OsGS1;2 and OsGS1;3) for cytosolic GS1.
   - Abnormal mRNA for **GS1**;1 was transcribed, and the **GS1** protein and its activity in the leaf blades were barely detectable in these mutants.
   - These findings suggested that the variation in **GS1** and NADH‚ÄêGOGAT protein contents in this population is related to the changes in the rate of nitrogen recycling from senescing organs to developing organs, leading to changes in these physiological traits.
   - **OsGLN1;1** accumulated in dermatogen, epidermis and exodermis under nitrogen-limited condition.
   - By contrast, OsGLN1;2 was abundantly expressed in the same cell layers under nitrogen-sufficient conditions, replenishing the loss of **OsGLN1;1** following ammonium treatment.
   - The results indicate that **GS1**;1 is important for normal growth and grain filling in rice; **GS1**;2 and **GS1**;3 were not able to compensate for **GS1**;1 function.
   - C-22 showed a 12-37% lower content of **GS1** protein in leaf blades than Koshihikari, which was in good agreement with a QTL region positively affected by the japonica chromosome.
   - KEY MESSAGE: Overexpression of **OsGS** gene modulates oxidative stress response in rice after exposure to cadmium stress.
   - Our results demonstrate that the co-overexpression of **OsGS1;1** and OsGS2 isoforms in transgenic rice plants enhanced its tolerance to osmotic and salinity stress at the seedling stage.
   - Concurrent Overexpression of **OsGS1;1** and OsGS2 Genes in Transgenic Rice (Oryza sativa L.): Impact on Tolerance to Abiotic Stresses..
   - Taken together, our results demonstrate that the concurrent overexpression of **OsGS1;1** and OsGS2 isoforms in rice enhanced physiological tolerance and agronomic performance under adverse abiotic stress conditions, apparently acting through multiple mechanistic routes.
   - GOLDEN2-LIKE transcription factor-encoding genes, which are involved in chloroplast biogenesis in rice, could not compensate for the lack of OsSIGs in the **OsGS1;1** mutant.
   - Microscopic analysis revealed mature chloroplast development in **OsGS1;1** roots but not in the roots of Osgs1;2, Osgs1;2-complemented lines, or the wildtype.
   - Thus, organic N assimilated by **OsGS1;1** affects a broad range of metabolites and transcripts involved in maintaining metabolic homeostasis and plastid development in rice roots, whereas OsGS1;2 has a more specific role, affecting mainly amino acid homeostasis but not C metabolism.
   - We observed (i) abnormal sugar and organic N accumulation and (ii) significant upregulation of genes associated with photosynthesis and chlorophyll biosynthesis in the roots of **OsGS1;1** but not Osgs1;2 knockout mutants.
   - Network analysis of the **OsGS1;1** mutant suggested that metabolism of glutamine was coordinated with the metabolic modules of sugar metabolism, tricarboxylic acid cycle, and carbon (C) fixation.
   - Alternative splicing of **OsGS1;1** affects nitrogen-use efficiency, grain development, and amylose content in rice..
   - The elite haplotype of **OsGS1;1** showed high **OsGS1;1**b activity, which improved NUE, affected grain development, and reduced amylose content.

## Function-related keywords:
   - [stem](/tags/stem/)&nbsp;,&nbsp;[seed](/tags/seed/)&nbsp;,&nbsp;[leaf](/tags/leaf/)&nbsp;,&nbsp;[nitrogen](/tags/nitrogen/)&nbsp;,&nbsp;[drought](/tags/drought/)&nbsp;,&nbsp;[root](/tags/root/)&nbsp;,&nbsp;[grain](/tags/grain/)&nbsp;,&nbsp;[growth](/tags/growth/)&nbsp;,&nbsp;[spikelet](/tags/spikelet/)&nbsp;,&nbsp;[sheath](/tags/sheath/)&nbsp;,&nbsp;[chloroplast](/tags/chloroplast/)&nbsp;,&nbsp;[grain-filling](/tags/grain-filling/)&nbsp;,&nbsp;[drought-tolerance](/tags/drought-tolerance/)&nbsp;,&nbsp;[cadmium](/tags/cadmium/)&nbsp;,&nbsp;[oxidative](/tags/oxidative/)&nbsp;,&nbsp;[seedling](/tags/seedling/)&nbsp;,&nbsp;[salinity](/tags/salinity/)&nbsp;,&nbsp;[tolerance](/tags/tolerance/)&nbsp;,&nbsp;[abiotic-stress](/tags/abiotic-stress/)&nbsp;,&nbsp;[stress](/tags/stress/)&nbsp;,&nbsp;[Al-tolerance](/tags/Al-tolerance/)&nbsp;,&nbsp;[biotic-stress](/tags/biotic-stress/)&nbsp;,&nbsp;[salinity-stress](/tags/salinity-stress/)&nbsp;,&nbsp;[transcription-factor](/tags/transcription-factor/)&nbsp;,&nbsp;[development](/tags/development/)&nbsp;,&nbsp;[photosynthesis](/tags/photosynthesis/)&nbsp;,&nbsp;[homeostasis](/tags/homeostasis/)&nbsp;,&nbsp;[chloroplast-development](/tags/chloroplast-development/)&nbsp;,&nbsp;[sugar](/tags/sugar/)&nbsp;,&nbsp;[amylose-content](/tags/amylose-content/)

## Literature:
   - [Regulation of glutamine synthetase isoforms in two differentially drought-tolerant rice (Oryza sativa L.) cultivars under water deficit conditions](https://www.doi.org/10.1007/s00299-012-1353-6)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1007/s00299-012-1353-6](https://www.doi.org/10.1007/s00299-012-1353-6)&nbsp;&nbsp;PMID:&nbsp;&nbsp;[23070303](https://pubmed.ncbi.nlm.nih.gov/23070303/)
   - [Influence of different nitrogen inputs on the members of ammonium transporter and glutamine synthetase genes in two rice genotypes having differential responsiveness to nitrogen](https://www.doi.org/10.1007/s11033-012-1650-8)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1007/s11033-012-1650-8](https://www.doi.org/10.1007/s11033-012-1650-8)&nbsp;&nbsp;PMID:&nbsp;&nbsp;[22531935](https://pubmed.ncbi.nlm.nih.gov/22531935/)
   - [Biochemical background and compartmentalized functions of cytosolic glutamine synthetase for active ammonium assimilation in rice roots](https://www.doi.org/10.1093/pcp/pch190)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1093/pcp/pch190](https://www.doi.org/10.1093/pcp/pch190)&nbsp;&nbsp;PMID:&nbsp;&nbsp;[15574840](https://pubmed.ncbi.nlm.nih.gov/15574840/)
   - [Mapping of QTLs associated with cytosolic glutamine synthetase and NADH-glutamate synthase in rice (Oryza sativa L.)](https://www.doi.org/)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[](https://www.doi.org/)&nbsp;&nbsp;PMID:&nbsp;&nbsp;[11432939](https://pubmed.ncbi.nlm.nih.gov/11432939/)
   - [Severe reduction in growth rate and grain filling of rice mutants lacking OsGS1;1, a cytosolic glutamine synthetase1;1](https://www.doi.org/10.1111/j.1365-313X.2005.02406.x)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1111/j.1365-313X.2005.02406.x](https://www.doi.org/10.1111/j.1365-313X.2005.02406.x)&nbsp;&nbsp;PMID:&nbsp;&nbsp;[15918879](https://pubmed.ncbi.nlm.nih.gov/15918879/)
   - [Metabolomics data reveal a crucial role of cytosolic glutamine synthetase 1;1 in coordinating metabolic balance in rice](https://www.doi.org/10.1111/j.1365-313X.2011.04506.x)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1111/j.1365-313X.2011.04506.x](https://www.doi.org/10.1111/j.1365-313X.2011.04506.x)&nbsp;&nbsp;PMID:&nbsp;&nbsp;[21255162](https://pubmed.ncbi.nlm.nih.gov/21255162/)
   - [Three cDNA sequences coding for glutamine synthetase polypeptides in Oryza sativa L](https://www.doi.org/10.1007/BF00027323)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1007/BF00027323](https://www.doi.org/10.1007/BF00027323)&nbsp;&nbsp;PMID:&nbsp;&nbsp;[2577497](https://pubmed.ncbi.nlm.nih.gov/2577497/)
   - [Identification and characterization of a QTL on chromosome 2 for cytosolic glutamine synthetase content and panicle number in rice](https://www.doi.org/10.1007/s00122-004-1828-0)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1007/s00122-004-1828-0](https://www.doi.org/10.1007/s00122-004-1828-0)&nbsp;&nbsp;PMID:&nbsp;&nbsp;[15549232](https://pubmed.ncbi.nlm.nih.gov/15549232/)
   - [Overexpression of the glutamine synthetase gene modulates oxidative stress response in rice after exposure to cadmium stress](https://www.doi.org/10.1007/s00299-013-1464-8)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1007/s00299-013-1464-8](https://www.doi.org/10.1007/s00299-013-1464-8)&nbsp;&nbsp;PMID:&nbsp;&nbsp;[23743654](https://pubmed.ncbi.nlm.nih.gov/23743654/)
   - [Concurrent Overexpression of OsGS1;1 and OsGS2 Genes in Transgenic Rice (Oryza sativa L.): Impact on Tolerance to Abiotic Stresses](https://www.doi.org/10.3389/fpls.2018.00786)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.3389/fpls.2018.00786](https://www.doi.org/10.3389/fpls.2018.00786)&nbsp;&nbsp;PMID:&nbsp;&nbsp;[29977247](https://pubmed.ncbi.nlm.nih.gov/29977247/)
   - [Cytosolic GLUTAMINE SYNTHETASE 1;1 modulates metabolism and chloroplast development in roots](https://www.doi.org/10.1104/pp.19.01118)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1104/pp.19.01118](https://www.doi.org/10.1104/pp.19.01118)&nbsp;&nbsp;PMID:&nbsp;&nbsp;[32024696](https://pubmed.ncbi.nlm.nih.gov/32024696/)
   - [The Amino Acid Transporter OsAAP4 Contributes to Rice Tillering and Grain Yield by Regulating Neutral Amino Acid Allocation through Two Splicing Variants](https://www.doi.org/10.1186/s12284-020-00446-9)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1186/s12284-020-00446-9](https://www.doi.org/10.1186/s12284-020-00446-9)&nbsp;&nbsp;PMID:&nbsp;&nbsp;[33409665](https://pubmed.ncbi.nlm.nih.gov/33409665/)
   - [Alternative splicing of OsGS1;1 affects nitrogen-use efficiency, grain development, and amylose content in rice](https://www.doi.org/10.1111/tpj.15768)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1111/tpj.15768](https://www.doi.org/10.1111/tpj.15768)&nbsp;&nbsp;PMID:&nbsp;&nbsp;[35404523](https://pubmed.ncbi.nlm.nih.gov/35404523/)

## Related News:

## Gene Resources:
- **NCBI ID:**  [NM_001054580.1](http://www.ncbi.nlm.nih.gov/nuccore/NM_001054580.1)&nbsp;,&nbsp;[AB037595](http://www.ncbi.nlm.nih.gov/nuccore/AB037595)&nbsp;,&nbsp;[X14245](http://www.ncbi.nlm.nih.gov/nuccore/X14245)
- **UniProt accessions:** [](https://www.uniprot.org/uniprotkb//entry)

## Sequences:
- **CDS Sequence:**
- **Protein Sequence:**
