---
description: LOC_Os01g20160 ; Os01g0307500 ; Oryza sativa
meta_title:
title: OsHKT1;5,SKC1,OsHKT8,OsHK1;5
categories: [Oryza sativa]
tags: [salinity stress, salinity, salt tolerance, homeostasis, xylem, leaf, shoot, transporter, salt stress, salt, vascular bundle, growth, tolerance, stress, reproductive, phloem, plasma membrane, development]
---

## Gene Details:
- **MSU gene ID:** [LOC_Os01g20160](http://rice.uga.edu/cgi-bin/ORF_infopage.cgi?orf=LOC_Os01g20160)  
- **RAPdb gene ID:** [Os01g0307500](https://rapdb.dna.affrc.go.jp/locus/?name=Os01g0307500)  
- **Gene Symbol:** OsHKT1;5,SKC1,OsHKT8,OsHK1;5
- **Gene Name:**
- **Genome:**  [MSU7](http://rice.uga.edu/)&nbsp;,&nbsp;[IRGSP-1.0](https://rapdb.dna.affrc.go.jp/download/irgsp1.html)
- **Species:** *Oryza sativa*

## Functional Descriptions:
   - Here we present a review on vital physiological functions of HKT transporters including AtHKT1;1 and **OsHKT1;5** in preventing shoot Na(+) over-accumulation by mediating Na(+) exclusion from xylem vessels in the presence of a large amount of Na(+) thereby protecting leaves from salinity stress.
   - We previously mapped a rice QTL, **SKC1**, that maintained K(+) homeostasis in the salt-tolerant variety under salt stress, consistent with the earlier finding that K(+) homeostasis is important in salt tolerance.
   - Physiological analysis suggested that **SKC1** is involved in regulating K(+)/Na(+) homeostasis under salt stress, providing a potential tool for improving salt tolerance in crops.
   - **SKC1** is preferentially expressed in the parenchyma cells surrounding the xylem vessels.
   - The association of leaf Na+ concentrations with cultivar-groups was very weak, but association with the **OsHKT1;5** allele was generally strong.
   - Seven major and three minor alleles of **OsHKT1;5** were identified, and their comparisons with the leaf Na+ concentration showed that the Aromatic allele conferred the highest exclusion and the Japonica allele the least.
   - In addition, lower expression of **OsHKT1;5** and OsSOS1 in old leaves may decrease frequency of retrieving Na+ from old leaf cells.
   - To understand the molecular basis of this QTL, we isolated the **SKC1** gene by map-based cloning and found that it encoded a member of HKT-type transporters.
   - Voltage-clamp analysis showed that **SKC1** protein functions as a Na(+)-selective transporter.
   - For **OsHKT1;5**, both transcript abundance and protein structural features within the selectivity filter could control shoot Na(+) accumulation in a range of rice varieties.
   - **OsHKT1;5** mediates Na(+) exclusion in the vasculature to protect leaf blades and reproductive tissues from salt toxicity in rice..
   - Additionally, direct introduction of (22) Na(+) tracer to leaf sheaths also demonstrated the involvement of **OsHKT1;5** in xylem Na(+) unloading in leaf sheaths.
   - Together with the characteristic (22) Na(+) allocation in the blade of the developing immature leaf in the mutants, these results suggest a novel function of **OsHKT1;5** in mediating Na(+) exclusion in the phloem to prevent Na(+) transfer to young leaf blades.
   - Our findings further demonstrate that the function of **OsHKT1;5** is crucial over growth stages of rice, including the protection of the next generation seeds as well as of vital leaf blades under salt stress.
   - Immuno-staining using an anti-**OsHKT1;5** peptide antibody indicated that **OsHKT1;5** is localized in cells adjacent to the xylem in roots.
   - Furthermore, **OsHKT1;5** was indicated to present in the plasma membrane and found to localize also in the phloem of diffuse vascular bundles in basal nodes.
   - Salt tolerance QTL analysis of rice has revealed that the SKC1 locus, which is involved in a higher K(+) /Na(+) ratio in shoots, corresponds to the **OsHKT1;5** gene encoding a Na(+) -selective transporter.
   - However, physiological roles of **OsHKT1;5** in rice exposed to salt stress remain elusive and no **OsHKT1;5** gene disruption mutants have been characterized to date.
   - Salt stress-induced increases in the **OsHKT1;5** transcript was observed in roots and basal stems including basal nodes.
   - We also report the development and validation of a new Cleavage Amplified Polymorphic Sequence (CAPS) marker (**OsHKT1;5**V395) that targets a codon in the sodium transporter gene **OsHKT1;5** (Saltol/SKC1 locus) that is associated with sodium transport rates in the above rice landraces.
   - Analysis of genetic diversity and population structure using SSR markers and validation of a Cleavage Amplified Polymorphic Sequences (CAPS) marker involving the sodium transporter **OsHKT1;5** in saline tolerant rice (Oryza sativa L.) landraces..
   - Allelic variation in rice **OsHKT1;5** sequence in specific landraces (Nona Bokra **OsHKT1;5**-NB/Nipponbare **OsHKT1;5**-Ni) correlates with variation in salt tolerance.
   - In rice, the **OsHKT1;5** gene has been reported to be a critical determinant of salt tolerance.
   - Changes in Expression Level of **OsHKT1;5** Alters Activity of Membrane Transporters Involved in K + and Ca 2+ Acquisition and Homeostasis in Salinized Rice Roots.
   - Furthermore, expression analysis suggested that OsGrx_C7 acted as positive regulator of salt tolerance by reinforcing the expression of transporters (OsHKT2;1, **OsHKT1;5** and OsSOS1) engaged in Na+ homeostasis in overexpressing plants.
   - Intriguingly, salt stress facilitates the nuclear relocation of OsDNAJ15 so that it can interact with OsBAG4, and OsDNAJ15 and OsBAG4 synergistically facilitate the DNA-binding activity of OsMYB106 to positively regulate the expression of **OsHKT1;5**.
   - Using a previously generated high-throughput activation tagging-based T-DNA insertion mutant pool, we isolated a mutant exhibiting salt stress-sensitive phenotype, caused by a reduction in **OsHKT1;5** transcripts.
   - Plasma membrane-localized Hsp40/DNAJ chaperone protein facilitates OsSUVH7-OsBAG4-OsMYB106 transcriptional complex formation for **OsHKT1;5** activation..
   - The real-time quantitative PCR (qRT-PCR) and transcriptome analysis revealed that OsWRKY54 regulated the expression of some essential genes related to salt tolerance, such as OsNHX4 and **OsHKT1;5**.

## Function-related keywords:
   - [salinity-stress](/tags/salinity-stress/)&nbsp;,&nbsp;[salinity](/tags/salinity/)&nbsp;,&nbsp;[salt-tolerance](/tags/salt-tolerance/)&nbsp;,&nbsp;[homeostasis](/tags/homeostasis/)&nbsp;,&nbsp;[xylem](/tags/xylem/)&nbsp;,&nbsp;[leaf](/tags/leaf/)&nbsp;,&nbsp;[shoot](/tags/shoot/)&nbsp;,&nbsp;[transporter](/tags/transporter/)&nbsp;,&nbsp;[salt-stress](/tags/salt-stress/)&nbsp;,&nbsp;[salt](/tags/salt/)&nbsp;,&nbsp;[vascular-bundle](/tags/vascular-bundle/)&nbsp;,&nbsp;[growth](/tags/growth/)&nbsp;,&nbsp;[tolerance](/tags/tolerance/)&nbsp;,&nbsp;[stress](/tags/stress/)&nbsp;,&nbsp;[reproductive](/tags/reproductive/)&nbsp;,&nbsp;[phloem](/tags/phloem/)&nbsp;,&nbsp;[plasma-membrane](/tags/plasma-membrane/)&nbsp;,&nbsp;[development](/tags/development/)

## Literature:
   - [QTLs for Na+ and K+ uptake of the shoots and roots controlling rice salt tolerance](https://www.doi.org/10.1007/s00122-003-1421-y)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1007/s00122-003-1421-y](https://www.doi.org/10.1007/s00122-003-1421-y)&nbsp;&nbsp;PMID:&nbsp;&nbsp;[14513218](https://pubmed.ncbi.nlm.nih.gov/14513218/)
   - [Effects of salt stress on ion balance and nitrogen metabolism of old and young leaves in rice (Oryza sativa L.)](https://www.doi.org/10.1186/1471-2229-12-194)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1186/1471-2229-12-194](https://www.doi.org/10.1186/1471-2229-12-194)&nbsp;&nbsp;PMID:&nbsp;&nbsp;[23082824](https://pubmed.ncbi.nlm.nih.gov/23082824/)
   - [Salinity tolerance, Na+ exclusion and allele mining of HKT1;5 in Oryza sativa and O. glaberrima: many sources, many genes, one mechanism?](https://www.doi.org/10.1186/1471-2229-13-32)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1186/1471-2229-13-32](https://www.doi.org/10.1186/1471-2229-13-32)&nbsp;&nbsp;PMID:&nbsp;&nbsp;[23445750](https://pubmed.ncbi.nlm.nih.gov/23445750/)
   - [A conserved primary salt tolerance mechanism mediated by HKT transporters: a mechanism for sodium exclusion and maintenance of high K(+)/Na(+) ratio in leaves during salinity stress](https://www.doi.org/10.1111/j.1365-3040.2009.02056.x)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1111/j.1365-3040.2009.02056.x](https://www.doi.org/10.1111/j.1365-3040.2009.02056.x)&nbsp;&nbsp;PMID:&nbsp;&nbsp;[19895406](https://pubmed.ncbi.nlm.nih.gov/19895406/)
   - [A rice quantitative trait locus for salt tolerance encodes a sodium transporter](https://www.doi.org/10.1038/ng1643)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1038/ng1643](https://www.doi.org/10.1038/ng1643)&nbsp;&nbsp;PMID:&nbsp;&nbsp;[16155566](https://pubmed.ncbi.nlm.nih.gov/16155566/)
   - [A two-staged model of Na+ exclusion in rice explained by 3D modeling of HKT transporters and alternative splicing](https://www.doi.org/10.1371/journal.pone.0039865)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1371/journal.pone.0039865](https://www.doi.org/10.1371/journal.pone.0039865)&nbsp;&nbsp;PMID:&nbsp;&nbsp;[22808069](https://pubmed.ncbi.nlm.nih.gov/22808069/)
   - [OsHKT1;5 mediates Na+ exclusion in the vasculature to protect leaf blades and reproductive tissues from salt toxicity in rice](https://www.doi.org/10.1111/tpj.13595)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1111/tpj.13595](https://www.doi.org/10.1111/tpj.13595)&nbsp;&nbsp;PMID:&nbsp;&nbsp;[28488420](https://pubmed.ncbi.nlm.nih.gov/28488420/)
   - [Analysis of genetic diversity and population structure using SSR markers and validation of a Cleavage Amplified Polymorphic Sequences (CAPS) marker involving the sodium transporter OsHKT1;5 in saline tolerant rice (Oryza sativa L.) landraces](https://www.doi.org/10.1016/j.gene.2019.143976)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1016/j.gene.2019.143976](https://www.doi.org/10.1016/j.gene.2019.143976)&nbsp;&nbsp;PMID:&nbsp;&nbsp;[31306715](https://pubmed.ncbi.nlm.nih.gov/31306715/)
   - [HKT1;5 Transporter Gene Expression and Association of Amino Acid Substitutions With Salt Tolerance Across Rice Genotypes](https://www.doi.org/10.3389/fpls.2019.01420)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.3389/fpls.2019.01420](https://www.doi.org/10.3389/fpls.2019.01420)&nbsp;&nbsp;PMID:&nbsp;&nbsp;[31749823](https://pubmed.ncbi.nlm.nih.gov/31749823/)
   - [Homology Modeling Identifies Crucial Amino-Acid Residues That Confer Higher Na+ Transport Capacity of OcHKT1;5 from Oryza coarctata Roxb](https://www.doi.org/10.1093/pcp/pcaa061)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1093/pcp/pcaa061](https://www.doi.org/10.1093/pcp/pcaa061)&nbsp;&nbsp;PMID:&nbsp;&nbsp;[32379873](https://pubmed.ncbi.nlm.nih.gov/32379873/)
   - [Changes in Expression Level of OsHKT1;5 Alters Activity of Membrane Transporters Involved in K+ and Ca2+ Acquisition and Homeostasis in Salinized Rice Roots](https://www.doi.org/10.3390/ijms21144882)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.3390/ijms21144882](https://www.doi.org/10.3390/ijms21144882)&nbsp;&nbsp;PMID:&nbsp;&nbsp;[32664377](https://pubmed.ncbi.nlm.nih.gov/32664377/)
   - [CC-type glutaredoxin, OsGrx_C7 plays a crucial role in enhancing protection against salt stress in rice](https://www.doi.org/10.1016/j.jbiotec.2021.02.008)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1016/j.jbiotec.2021.02.008](https://www.doi.org/10.1016/j.jbiotec.2021.02.008)&nbsp;&nbsp;PMID:&nbsp;&nbsp;[33610657](https://pubmed.ncbi.nlm.nih.gov/33610657/)
   - [Clock component OsPRR73 positively regulates rice salt tolerance by modulating OsHKT2;1-mediated sodium homeostasis](https://www.doi.org/10.15252/embj.2020105086)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.15252/embj.2020105086](https://www.doi.org/10.15252/embj.2020105086)&nbsp;&nbsp;PMID:&nbsp;&nbsp;[33347628](https://pubmed.ncbi.nlm.nih.gov/33347628/)
   - [Plasma membrane-localized Hsp40/DNAJ chaperone protein facilitates OsSUVH7-OsBAG4-OsMYB106 transcriptional complex formation for OsHKT1;5 activation](https://www.doi.org/10.1111/jipb.13403)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1111/jipb.13403](https://www.doi.org/10.1111/jipb.13403)&nbsp;&nbsp;PMID:&nbsp;&nbsp;[36349953](https://pubmed.ncbi.nlm.nih.gov/36349953/)
   - [The WRKY Transcription Factor OsWRKY54 Is Involved in Salt Tolerance in Rice](https://www.doi.org/10.3390/ijms231911999)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.3390/ijms231911999](https://www.doi.org/10.3390/ijms231911999)&nbsp;&nbsp;PMID:&nbsp;&nbsp;[36233306](https://pubmed.ncbi.nlm.nih.gov/36233306/)
   - [Natural variations of chlorophyll fluorescence and ion transporter genes influenced the differential response of japonica rice germplasm with different salt tolerances](https://www.doi.org/10.3389/fpls.2023.1095929)&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.3389/fpls.2023.1095929](https://www.doi.org/10.3389/fpls.2023.1095929)&nbsp;&nbsp;PMID:&nbsp;&nbsp;[37008489](https://pubmed.ncbi.nlm.nih.gov/37008489/)

## Related News:

## Gene Resources:
- **NCBI ID:**  [DQ148410](http://www.ncbi.nlm.nih.gov/nuccore/DQ148410)
- **UniProt accessions:** [](https://www.uniprot.org/uniprotkb//entry)

## Sequences:
- **CDS Sequence:**
- **Protein Sequence:**
