---
description: LOC_Os08g39890 ; Os08g0509600 ; Oryza sativa
meta_title:
title: OsSPL14;IPA1
categories: [Oryza sativa]
tags: [yield, lodging, tiller, architecture, lodging resistance, growth, breeding, grain, branching, shoot, tiller number, vegetative, grain yield, panicle architecture, panicle, erect, reproductive, tillering, height, leaf, nitrogen, development, transcription activator, plant growth, transcription factor, strigolactone, resistance, defense, disease, disease resistance, magnaporthe oryzae, immunity, pathogen, gibberellin, Gibberellin, seedling, seed, starch, seed germination,  ga , GA, seedling growth, seedlings, root, drought, tolerance, ABA, drought tolerance, stomatal, abscisic acid, plant architecture, drought resistance, ABA biosynthesis, ABA catabolism,  ABA , grains per panicle, stress tolerance, reactive oxygen species]
---

## Gene Details:
- **MSU gene ID:** [LOC_Os08g39890](http://rice.uga.edu/cgi-bin/ORF_infopage.cgi?orf=LOC_Os08g39890)  
- **RAPdb gene ID:** [Os08g0509600](https://rapdb.dna.affrc.go.jp/locus/?name=Os08g0509600)  
- **Gene Symbol:** OsSPL14;IPA1;WFP
- **Gene Name:**
- **Genome:**  [MSU7](http://rice.uga.edu/),&nbsp;&nbsp;[IRGSP-1.0](https://rapdb.dna.affrc.go.jp/download/irgsp1.html)
- **Species:** *Oryza sativa*

## Functional Descriptions:
   - Two studies describe how regulatory variation at the rice gene **OsSPL14** can lead to altered plant morphology and improve grain yield.
   - We demonstrate that a point mutation in **OsSPL14** perturbs OsmiR156-directed regulation of **OsSPL14**, generating an 'ideal' rice plant with a reduced tiller number, increased lodging resistance and enhanced grain yield.
   - Moreover, our results demonstrated that **IPA1** could directly bind to the promoter of rice teosinte branched1, a negative regulator of tiller bud outgrowth, to suppress rice tillering, and directly and positively regulate dense and erect panicle1, an important gene regulating panicle architecture, to influence plant height and panicle length.
   - Here, we report the cloning and characterization of a semidominant quantitative trait locus, **IPA1** (Ideal Plant Architecture 1), which profoundly changes rice plant architecture and substantially enhances rice grain yield.
   - We also demonstrate the feasibility of using the OsSLP14(**WFP**) allele to increase rice crop yield.
   - Introduction of the high-yielding **OsSPL14**(WFP) allele into the standard rice variety Nipponbare resulted in increased rice production.
   - Ideal plant architecture1 (**IPA1**) is critical in regulating rice (Oryza sativa) plant architecture and substantially enhances grain yield.
   - The elucidation of target genes of **IPA1** genome-wide will contribute to understanding the molecular mechanisms underlying plant architecture and to facilitating the breeding of elite varieties with ideal plant architecture.
   - Higher expression of **OsSPL14** in the reproductive stage promotes panicle branching and higher grain yield in rice.
   - **OsSPL14** promotes panicle branching and higher grain productivity in rice.
   - **OsSPL14** controls shoot branching in the vegetative stage and is affected by microRNA excision.
   - Our study suggests that **OsSPL14** may help improve rice grain yield by facilitating the breeding of new elite rice varieties.
   - Here we show that the quantitative trait locus WFP (WEALTHY FARMER'S PANICLE) encodes **OsSPL14** (SQUAMOSA PROMOTER BINDING PROTEIN-LIKE 14, also known as IPA1).
   - To elucidate its molecular basis, we first confirmed **IPA1** as a functional transcription activator and then identified 1067 and 2185 genes associated with **IPA1** binding sites in shoot apices and young panicles, respectively, through chromatin immunoprecipitation sequencing assays.
   - Regulation of **OsSPL14** by OsmiR156 defines ideal plant architecture in rice.
   - Expression analysis of **OsSPL14** (LOC_Os08g39890) gene reported to be associated with increased panicle branching and higher grain yield through real time PCR in leaf and three stages of panicle has shown differential temporal expression and its association with yield and yield related components across the genotypes.
   - Since **OsSPL14** is a functional transcription activator, its association of expression in leaf and three panicle stages with yield components as observed in the present study suggests the role of nitrogen metabolism related genes in plant growth and development and its conversion into yield components in rice.
   - The expression of **OsSPL14** at panicle stage 3, has shown correlation (P&lt;0.
   - Enhanced expression of **OsSPL14** gene and its association with yield components in rice (Oryza sativa) under low nitrogen conditions.
   - **IPA1** functions as a downstream transcription factor repressed by D53 in strigolactone signaling in rice.
   - These findings reveal that **IPA1** is likely one of the long-speculated transcription factors that act with D53 to mediate the SL-regulated tiller development in rice.
   - **IPA1** returns to a nonphosphorylated state within 48 hours after infection, resuming support of the growth needed for high yield.
   - Thus, **IPA1** promotes both yield and disease resistance by sustaining a balance between growth and immunity.
   - Phosphorylated **IPA1** binds to the promoter of the pathogen defense gene WRKY45 and activates its expression, leading to enhanced disease resistance.
   - Mechanistically, phosphorylation of **IPA1** at amino acid Ser163 within its DNA binding domain occurs in response to infection by the fungus Magnaporthe oryzae and alters the DNA binding specificity of **IPA1**.
   - Here we report that higher **IPA1** levels enhance immunity.
   - We discovered that gibberellin signalling might be partially responsible for the disease resistance and developmental defects in **IPA1** overexpressors.
   - We then generated transgenic rice plants expressing **IPA1** with the pathogen-inducible promoter of OsHEN1; these plants had both enhanced disease resistance and enhanced yield-related traits.
   - Using transient dual-luciferase and yeast one-hybrid assays, **IPA1** was found to directly activate the expression of OsWRKY51 and OsWRKY71, which would interfere with the binding affinity of GA-induced transcription factor OsGAMYB to inhibit the expression of <a6><c1>-amylase genes.
   - To investigate the role of **IPA1** in early seedling development, we developed a pair of **IPA1**/ipal-NILs and found that seed germination and early seedling growth were retarded in the **IPA1**-NIL.
   - In summary, our results suggest that **IPA1** negatively regulates seed germination and early seedling growth by interfering with starch metabolism via the GA and WRKY pathways.
   - **IPA1** Negatively Regulates Early Rice Seedling Development by Interfering with Starch Metabolism via the GA and WRKY Pathways.
   - The **IPA1** plants had a better-developed root system and smaller leaf stomatal aperture.
   - IPA1 (IDEAL PLANT ARCHITECTURE 1)/**OsSPL14** encodes a transcription factor and has been reported to function in both rice ideal plant architecture and biotic resistance.
   - In addition, based on yeast one-hybrid assay and dual-luciferase assay, **IPA1** was found to directly activate the promoter activity of OsHOX12, a transcription factor promoting ABA biosynthesis, and OsNAC52, a positive regulator of the ABA pathway.
   - In both the control and PEG-treated conditions, ABA content in the **IPA1** seedlings was significantly higher than that in the **IPA1** seedlings.
   - Expression of the ABA biosynthesis genes was detected to be up-regulated, whereas the expression of ABA catabolism genes was down-regulated in the **IPA1** seedlings.
   - Combined with the previous studies, our results suggested that **IPA1** could improve rice seedling drought tolerance mainly through activating the ABA pathway and that regulation of the **IPA1**-mediated ABA pathway will be an important strategy for improving drought resistance of rice.
   - **IPA1** improves rice drought tolerance at seedling stage mainly through activating abscisic acid pathway.
   - Here, with a pair of **IPA1** and **IPA1**-NILs (Near Iso-genic Lines), we found that **IPA1** could significantly improve rice drought tolerance at seedling stage.
   - KEY MESSAGE: **IPA1** enhances rice drought tolerance mainly through activating the ABA pathway.
   - Mechanistic studies revealed that the deleted fragment is a target site for the transcription factor An-1 to repress **IPA1** expression in panicles and roots.
   - In this study, we identified a 54-base pair cis-regulatory region in **IPA1** via a tiling-deletion-based CRISPR-Cas9 screen that, when deleted, resolves the tradeoff between grains per panicle and tiller number, leading to substantially enhanced grain yield per plant.
   - Crop genetic improvement requires balancing complex tradeoffs caused by gene pleiotropy and linkage drags, as exemplified by **IPA1** (Ideal Plant Architecture 1), a typical pleiotropic gene in rice that increases grains per panicle but reduces tillers.
   - **IPA1** improves drought tolerance by activating SNAC1 in rice.
   - Together, these findings indicated that the **IPA1** played important roles in drought tolerance by regulating SNAC1, thus activating the antioxidant system in rice.
   - Further investigation indicated that the **IPA1** plays a positive role in drought-stress tolerance by inducing reactive oxygen species scavenging in rice.
   - The **IPA1** is constitutively expressed and regulated by H(2)O(2), abscisic acid, NaCl and polyethylene glycol 6000 treatments in rice.

## Function-related keywords:
   - [yield](/tags/yield/),&nbsp;&nbsp;[lodging](/tags/lodging/),&nbsp;&nbsp;[tiller](/tags/tiller/),&nbsp;&nbsp;[architecture](/tags/architecture/),&nbsp;&nbsp;[lodging-resistance](/tags/lodging-resistance/),&nbsp;&nbsp;[growth](/tags/growth/),&nbsp;&nbsp;[breeding](/tags/breeding/),&nbsp;&nbsp;[grain](/tags/grain/),&nbsp;&nbsp;[branching](/tags/branching/),&nbsp;&nbsp;[shoot](/tags/shoot/),&nbsp;&nbsp;[tiller-number](/tags/tiller-number/),&nbsp;&nbsp;[vegetative](/tags/vegetative/),&nbsp;&nbsp;[grain-yield](/tags/grain-yield/),&nbsp;&nbsp;[panicle-architecture](/tags/panicle-architecture/),&nbsp;&nbsp;[panicle](/tags/panicle/),&nbsp;&nbsp;[erect](/tags/erect/),&nbsp;&nbsp;[reproductive](/tags/reproductive/),&nbsp;&nbsp;[tillering](/tags/tillering/),&nbsp;&nbsp;[height](/tags/height/),&nbsp;&nbsp;[leaf](/tags/leaf/),&nbsp;&nbsp;[nitrogen](/tags/nitrogen/),&nbsp;&nbsp;[development](/tags/development/),&nbsp;&nbsp;[transcription-activator](/tags/transcription-activator/),&nbsp;&nbsp;[plant-growth](/tags/plant-growth/),&nbsp;&nbsp;[transcription-factor](/tags/transcription-factor/),&nbsp;&nbsp;[strigolactone](/tags/strigolactone/),&nbsp;&nbsp;[resistance](/tags/resistance/),&nbsp;&nbsp;[defense](/tags/defense/),&nbsp;&nbsp;[disease](/tags/disease/),&nbsp;&nbsp;[disease-resistance](/tags/disease-resistance/),&nbsp;&nbsp;[magnaporthe-oryzae](/tags/magnaporthe-oryzae/),&nbsp;&nbsp;[immunity](/tags/immunity/),&nbsp;&nbsp;[pathogen](/tags/pathogen/),&nbsp;&nbsp;[gibberellin](/tags/gibberellin/),&nbsp;&nbsp;[Gibberellin](/tags/Gibberellin/),&nbsp;&nbsp;[seedling](/tags/seedling/),&nbsp;&nbsp;[seed](/tags/seed/),&nbsp;&nbsp;[starch](/tags/starch/),&nbsp;&nbsp;[seed-germination](/tags/seed-germination/),&nbsp;&nbsp;[ga](/tags/ga/),&nbsp;&nbsp;[GA](/tags/GA/),&nbsp;&nbsp;[seedling-growth](/tags/seedling-growth/),&nbsp;&nbsp;[seedlings](/tags/seedlings/),&nbsp;&nbsp;[root](/tags/root/),&nbsp;&nbsp;[drought](/tags/drought/),&nbsp;&nbsp;[tolerance](/tags/tolerance/),&nbsp;&nbsp;[ABA](/tags/ABA/),&nbsp;&nbsp;[drought-tolerance](/tags/drought-tolerance/),&nbsp;&nbsp;[stomatal](/tags/stomatal/),&nbsp;&nbsp;[abscisic-acid](/tags/abscisic-acid/),&nbsp;&nbsp;[plant-architecture](/tags/plant-architecture/),&nbsp;&nbsp;[drought-resistance](/tags/drought-resistance/),&nbsp;&nbsp;[ABA-biosynthesis](/tags/ABA-biosynthesis/),&nbsp;&nbsp;[ABA-catabolism](/tags/ABA-catabolism/),&nbsp;&nbsp;[grains-per-panicle](/tags/grains-per-panicle/),&nbsp;&nbsp;[stress-tolerance](/tags/stress-tolerance/),&nbsp;&nbsp;[reactive-oxygen-species](/tags/reactive-oxygen-species/)

## Literature:
   - [Regulation of OsSPL14 by OsmiR156 defines ideal plant architecture in rice](https://www.doi.org/10.1038/ng.591).&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1038/ng.591](https://www.doi.org/10.1038/ng.591);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[20495565](https://pubmed.ncbi.nlm.nih.gov/20495565/)
   - [OsSPL14 promotes panicle branching and higher grain productivity in rice](https://www.doi.org/10.1038/ng.592).&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1038/ng.592](https://www.doi.org/10.1038/ng.592);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[20495564](https://pubmed.ncbi.nlm.nih.gov/20495564/)
   - [Shaping a better rice plant](https://www.doi.org/10.1038/ng0610-475).&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1038/ng0610-475](https://www.doi.org/10.1038/ng0610-475);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[20502488](https://pubmed.ncbi.nlm.nih.gov/20502488/)
   - [Genome-wide binding analysis of the transcription activator ideal plant architecture1 reveals a complex network regulating rice plant architecture](https://www.doi.org/10.1105/tpc.113.113639).&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1105/tpc.113.113639](https://www.doi.org/10.1105/tpc.113.113639);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[24170127](https://pubmed.ncbi.nlm.nih.gov/24170127/)
   - [Enhanced expression of OsSPL14 gene and its association with yield components in rice (Oryza sativa) under low nitrogen conditions](https://www.doi.org/10.1016/j.gene.2015.10.062).&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1016/j.gene.2015.10.062](https://www.doi.org/10.1016/j.gene.2015.10.062);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[26519999](https://pubmed.ncbi.nlm.nih.gov/26519999/)
   - [A natural tandem array alleviates epigenetic repression of IPA1 and leads to superior yielding rice](https://www.doi.org/10.1038/ncomms14789).&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1038/ncomms14789](https://www.doi.org/10.1038/ncomms14789);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[28317902](https://pubmed.ncbi.nlm.nih.gov/28317902/)
   - [IPA1 functions as a downstream transcription factor repressed by D53 in strigolactone signaling in rice](https://www.doi.org/10.1038/cr.2017.102).&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1038/cr.2017.102](https://www.doi.org/10.1038/cr.2017.102);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[28809396](https://pubmed.ncbi.nlm.nih.gov/28809396/)
   - [Introgression of a functional epigenetic OsSPL14WFP allele into elite indica rice genomes greatly improved panicle traits and grain yield](https://www.doi.org/10.1038/s41598-018-21355-4).&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1038/s41598-018-21355-4](https://www.doi.org/10.1038/s41598-018-21355-4);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[29497052](https://pubmed.ncbi.nlm.nih.gov/29497052/)
   - [A single transcription factor promotes both yield and immunity in rice](https://www.doi.org/10.1126/science.aat7675).&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1126/science.aat7675](https://www.doi.org/10.1126/science.aat7675);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[30190406](https://pubmed.ncbi.nlm.nih.gov/30190406/)
   - [Inducible overexpression of Ideal Plant Architecture1 improves both yield and disease resistance in rice](https://www.doi.org/10.1038/s41477-019-0383-2).&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1038/s41477-019-0383-2](https://www.doi.org/10.1038/s41477-019-0383-2);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[30886331](https://pubmed.ncbi.nlm.nih.gov/30886331/)
   - [MiR529a controls plant height, tiller number, panicle architecture and grain size by regulating SPL target genes in rice (Oryza sativa L.)](https://www.doi.org/10.1016/j.plantsci.2020.110728).&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1016/j.plantsci.2020.110728](https://www.doi.org/10.1016/j.plantsci.2020.110728);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[33288029](https://pubmed.ncbi.nlm.nih.gov/33288029/)
   - [IPA1 Negatively Regulates Early Rice Seedling Development by Interfering with Starch Metabolism via the GA and WRKY Pathways](https://www.doi.org/10.3390/ijms22126605).&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.3390/ijms22126605](https://www.doi.org/10.3390/ijms22126605);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[34203082](https://pubmed.ncbi.nlm.nih.gov/34203082/)
   - [ipa1 improves rice drought tolerance at seedling stage mainly through activating abscisic acid pathway](https://www.doi.org/10.1007/s00299-021-02804-3).&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1007/s00299-021-02804-3](https://www.doi.org/10.1007/s00299-021-02804-3);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[34694441](https://pubmed.ncbi.nlm.nih.gov/34694441/)
   - [Osa-miR535 targets SQUAMOSA promoter binding protein-like 4 to regulate blast disease resistance in rice](https://www.doi.org/10.1111/tpj.15663).&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1111/tpj.15663](https://www.doi.org/10.1111/tpj.15663);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[34997660](https://pubmed.ncbi.nlm.nih.gov/34997660/)
   - [Targeting a gene regulatory element enhances rice grain yield by decoupling panicle number and size](https://www.doi.org/10.1038/s41587-022-01281-7).&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1038/s41587-022-01281-7](https://www.doi.org/10.1038/s41587-022-01281-7);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[35449414](https://pubmed.ncbi.nlm.nih.gov/35449414/)
   - [IPA1 improves drought tolerance by activating SNAC1 in rice](https://www.doi.org/10.1186/s12870-023-04062-9).&nbsp;&nbsp;DOI:&nbsp;&nbsp;[10.1186/s12870-023-04062-9](https://www.doi.org/10.1186/s12870-023-04062-9);&nbsp;&nbsp;PMID:&nbsp;&nbsp;[36698063](https://pubmed.ncbi.nlm.nih.gov/36698063/)

## Related News:

## Gene Resources:
- **NCBI ID:**  [GU136674](http://www.ncbi.nlm.nih.gov/nuccore/GU136674)
- **UniProt accessions:** [](https://www.uniprot.org/uniprotkb//entry)

## Sequences:
- **CDS Sequence:**
- **Protein Sequence:**
