---
title: "Gossypium arboreum (A2) 'SXY1' genome HAU_v1"
date: 2021-08-08T18:48:15+08:00
meta_title: ""
description: "this is meta description"
image: "/images/7.png"
categories: ["Application", "Data"]
author: "Wang, M. et al"
tags: ["nextjs", "tailwind"]
draft: false
---
## Overview
### Analysis Name:Gossypium arboreum (A2) 'SXY1' genome HAU_v1
### Method:llumina and PacBio
### Source:(v1)
### About the assembly:
In this study, we applied Oxford Nanopore sequencing technology to assemble G. rotundifolium (K2*) 'K201', G. arboreum (A2) 'SXY1', and G. raimondii (D5) 'D502' genomes. G. arboreum and G. raimondii genomes have been de novo assembled previously using Illumina and PacBio reads, but both genomes have a number of sequence gaps and require an improvement in assembly contiguity. We generated a total of 304 Gb, 212 Gb, 125 Gb Nanopore sequencing data with a genome coverage 124×, 131×, 167× for K2*, A2 and D5, respectively. We assembled 3,593, 1,173 and 366 contigs for G. rotundifolium, G. arboreum and G. raimondii with a contig length of 2.44 Gb, 1.62 Gb and 0.75 Gb, respectively (Table 1). These initial contigs were polished using Illumina paired-end reads with a genome coverage of 108×, 118×, 132× for K2*, A2 and D5. The contig N50 is 5.33 Mb, 11.69 Mb and 17.04 Mb for K2*, A2 and D5, respectively. The maximum contig has a length of 32.72 Mb, 58.57 Mb and 43.74 Mb. After polishing contig using Illumina reads, we used high-through chromosome conformation capture (Hi-C) data to order and orient contigs, aimed at constructing pseudo chromosomes of each species. In the Hi-C assisted assembly, 2,559, 485 and 201 contigs were placed on the 13 chromosomes of K2*, A2 and D5 genomes, occupying over 99% of genome length.

**_*Should be K12_**

### Table 1. Summary of genome assemblies and annotations of G. rotundifolium, G. arboreum and G.raimondii.

Genomic feature|G. rotundifolium 'Grot K201'|G. arboreum 'Shixiya1'	|G. raimondii 'Grai D502'
|:-----:|:-----:|:-----:|:-----:|
Total length of contigs, bp|2,444,364,209|	1,621,008,062	|750,197,587
Total length of scaffolds, bp|2,444,484,509	|1,621,030,562	|750,205,487
Total length of gaps, bp|120,300|	22,500	|7,900
Percentage of anchoring|99.28%|	99.47%	|99.57%
Percentage of anchoring and ordering|93.16%|	98.84%|	99.01%
Number of contigs|3,593|	1,173|	366
Number of scaffolds|2,390|	948|	287
Contig N50, bp|	5,326,689|	11,691,474|	17,043,680
Contig N90, bp|	621,066|	2,910,421|	3,537,560
Scaffold N50, bp|	177,839,665|129,592,444	|57,716,579
Scaffold N90, bp|	115,394,628	|93,157,762	|49,929,625
Maximun contig length, bp|	32,728,186|	58,575,076|	43,739,617
Maximum scaffold length, bp|	205,722,655	|143,367,608|	63,188,200
GC content|	36.38%	|35.16%	|33.23%
Percentage of repeat sequences|	80.92%|	68.05%	|57.04%
GC content|	36.38%|	35.16%	|33.23%
Number of genes|41,590|	41,778|	40,820
 

### Supplementary Table 4. Comparing A2 genome with previously published genome version.

Genomic feature|HAU_A2|WHU_A2|CRI_A2
:-----:|:-----:|:-----:|:-----:|
Total assemblied size, bp|1,621,008,062|1,636,985,834|1,710,104,083
Number of total scaffolds|948|1,269|4,516
Total length of gaps, bp|22,500|116,300|3,730,000
Contig N50, bp|11,691,474|1,832,000|1,100,000
Scaffold N50, bp|129,592,444|118,841,821|113,035,596
Scaffold N90, bp|93,157,762|87,096,684|162,124
Percentage of anchoring and ordering|98.84%|92.19%|85.94%
Number of genes|41,778|43,278|40,960
 

## Publication

>[Wang, M. et al. Comparative genome analyses highlight transposon-mediated genome expansion and the evolutionary architecture of 3D genomic folding in cotton. Mol Biol Evol. 2021 May 11:msab128. doi: 10.1093/molbev/msab128.](https://academic.oup.com/mbe/article/38/9/3621/6262643?login=false)