<!doctype html>
<html
  itemscope
  class=""
  lang="en-us"
  itemtype="http://schema.org/WebPage">
  <head><script src="/livereload.js?mindelay=10&amp;v=2&amp;port=1313&amp;path=livereload" data-no-instant defer></script>
    
    <meta charset="utf-8" />


<meta
  name="viewport"
  content="width=device-width, initial-scale=1, maximum-scale=5" />


<meta name="theme-name" content="hugoplate" />









<link rel="manifest" href="/manifest.webmanifest" />
<meta
  name="msapplication-TileColor"
  content="#ddd" />
<meta
  name="theme-color"
  content="#ffffff" />






















  <base href="//localhost:1313/genes1/gossypium-arboreum-a2-sxy1-genome-hau_v1/" />







  


<title>Gossypium arboreum (A2) &#39;SXY1&#39; genome HAU_v1</title>















  <meta
    name="keywords"
    content="Boilerplate, Hugo" />




<meta
  name="description"
  content="this is meta description" />



  <meta name="author" content="zeon.studio" />






  
  






  







  

  
  
  


  
  


  





<meta property="og:title" content="Gossypium arboreum (A2) &#39;SXY1&#39; genome HAU_v1" />
<meta property="og:description" content="this is meta description" />
<meta property="og:type" content="website" />
<meta property="og:url" content="//localhost:1313/genes1/gossypium-arboreum-a2-sxy1-genome-hau_v1/" />


<meta name="twitter:title" content="Gossypium arboreum (A2) &#39;SXY1&#39; genome HAU_v1" />
<meta name="twitter:description" content="this is meta description" />





















<script>
  let indexURL = "//localhost:1313/searchindex.json";
  let includeSectionsInSearch = ["genes1","genes","pages","sections","submit"];
  let no_results_for = "No results for";
  let empty_search_results_placeholder = "Type something to search..";
</script>




















    
    
<meta http-equiv="x-dns-prefetch-control" content="on" />
<link rel="preconnect" href="https://use.fontawesome.com" crossorigin />
<link rel="preconnect" href="//cdnjs.cloudflare.com" />
<link rel="preconnect" href="//www.googletagmanager.com" />
<link rel="preconnect" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="https://use.fontawesome.com" />
<link rel="dns-prefetch" href="//ajax.googleapis.com" />
<link rel="dns-prefetch" href="//cdnjs.cloudflare.com" />
<link rel="dns-prefetch" href="//www.googletagmanager.com" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//connect.facebook.net" />
<link rel="dns-prefetch" href="//platform.linkedin.com" />
<link rel="dns-prefetch" href="//platform.twitter.com" />




<link rel="preconnect" href="https://fonts.googleapis.com" />
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
<script>
  (function () {
    const googleFont = document.createElement("link");
    googleFont.href = "https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Montserrat:wght@500;700&display=swap";
    googleFont.type = "text/css";
    googleFont.rel = "stylesheet";
    document.head.appendChild(googleFont);
  })();
</script>





  
    
      
    
  

  
    
      
    
  

  
    
      
    
  

  
    
      
    
  

  
    
      
    
  











  
  



<link
  href="/css/style.css"
  integrity=""
  rel="stylesheet" />


<link
  defer
  async
  rel="stylesheet"
  href="/css/style-lazy.css"
  integrity=""
  media="print"
  onload="this.media='all'; this.onload=null;" />

  </head>

  <body style="padding-top: 2px; padding-bottom: 2px;">
    
    
      



      

    
  
    
    <header
  class="header sticky top-0 z-30">
  <nav class="navbar container">
    
    <div class="order-0">
      
      <a class="navbar-brand block" href="/">
        






















  
  COTTON


      </a>
    </div>
    
    <input id="nav-toggle" type="checkbox" class="hidden" />
    <label
      for="nav-toggle"
      class="order-3 cursor-pointer flex items-center lg:hidden text-dark dark:text-white lg:order-1">
      <svg id="show-button" class="h-6 fill-current block" viewBox="0 0 20 20">
        <title>Menu Open</title>
        <path d="M0 3h20v2H0V3z m0 6h20v2H0V9z m0 6h20v2H0V0z"></path>
      </svg>
      <svg id="hide-button" class="h-6 fill-current hidden" viewBox="0 0 20 20">
        <title>Menu Close</title>
        <polygon
          points="11 9 22 9 22 11 11 11 11 22 9 22 9 11 -2 11 -2 9 9 9 9 -2 11 -2"
          transform="rotate(45 10 10)"></polygon>
      </svg>
    </label>
    

    
    <ul
      id="nav-menu"
      class="navbar-nav order-3 hidden lg:flex w-full pb-6 lg:order-1 lg:w-auto lg:space-x-2 lg:pb-0 xl:space-x-8">
      
      
        
        
        
        
          <li class="nav-item">
            <a
              class="nav-link "
              
              href="/FAS/"
              >FAS</a
            >
          </li>
        
      
        
        
        
        
          <li class="nav-item">
            <a
              class="nav-link "
              
              href="/genes/"
              >GENES</a
            >
          </li>
        
      
        
        
        
        
          <li class="nav-item">
            <a
              class="nav-link "
              
              href="/tags/"
              >FUNCTION</a
            >
          </li>
        
      
        
        
        
        
          <li class="nav-item nav-dropdown group relative">
            <span
              class="nav-link 
                
                
                
              
                
                
                
               inline-flex items-center">
              HELP
              <svg class="h-4 w-4 fill-current" viewBox="0 0 20 20">
                <path
                  d="M9.293 12.95l.707.707L15.657 8l-1.414-1.414L10 10.828 5.757 6.586 4.343 8z" />
              </svg>
            </span>
            <ul
              class="nav-dropdown-list lg:group-hover:visible lg:group-hover:opacity-100">
              
                
                
                <li class="nav-dropdown-item">
                  <a
                    class="nav-dropdown-link "
                    
                    href="/about/">
                    LINKS
                  </a>
                </li>
              
                
                
                <li class="nav-dropdown-item">
                  <a
                    class="nav-dropdown-link "
                    
                    href="/submit/">
                    SUBMIT
                  </a>
                </li>
              
            </ul>
          </li>
        
      
      
        <li class="mt-4 inline-block lg:hidden">
          <a
            class="btn btn-outline-primary btn-sm"
            href="/submit">
            Submit
          </a>
        </li>
      
    </ul>

    
     

  <div class="order-1 ml-auto flex items-center md:order-2 lg:ml-0" style="margin-right: 20px;">

    <input
  id="search-modal-input"
  type="text"
  placeholder="Search"
  autocomplete="on"
  aria-label="Search"
  aria-describedby="Search"
  placeholder="Search Post ..."
  style="width: 120px; /* 设置宽度为800像素 */
         padding: 5px;
         border: 2px solid #ccc;
         border-radius: 20px;
         box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
         outline: none;
         transition: border-color 0.3s ease-in-out;
         margin-right: 20px; /* 添加右边距 */"/>
    <button
        id="search-button"
        aria-label="search"
        class="border-border text-dark hover:text-primary dark:border-darkmode-border mr-5 inline-block border-r pr-5 text-xl dark:text-white dark:hover:text-darkmode-primary"
        data-target="search-modal"
        onclick="search()">
        <i class="fa-solid fa-search"></i>
    </button>

   
   </div>

    <div class="order-1 ml-auto flex items-center md:order-2 lg:ml-0" style="margin-right: 20px;"></div>
          









          


  <div class="theme-switcher mr-5">
    <input id="theme-switcher" data-theme-switcher type="checkbox" />
    <label for="theme-switcher">
      <span class="sr-only">theme switcher</span>
      <span>
        
        <svg
          class="absolute left-[4px] top-[4px] z-10 opacity-100 dark:opacity-0"
          viewBox="0 0 56 56"
          fill="#fff"
          height="16"
          width="16">
          <path
            d="M30 4.6c0-1-.9-2-2-2a2 2 0 0 0-2 2v5c0 1 .9 2 2 2s2-1 2-2Zm9.6 9a2 2 0 0 0 0 2.8c.8.8 2 .8 2.9 0L46 13a2 2 0 0 0 0-2.9 2 2 0 0 0-3 0Zm-26 2.8c.7.8 2 .8 2.8 0 .8-.7.8-2 0-2.9L13 10c-.7-.7-2-.8-2.9 0-.7.8-.7 2.1 0 3ZM28 16a12 12 0 0 0-12 12 12 12 0 0 0 12 12 12 12 0 0 0 12-12 12 12 0 0 0-12-12Zm23.3 14c1.1 0 2-.9 2-2s-.9-2-2-2h-4.9a2 2 0 0 0-2 2c0 1.1 1 2 2 2ZM4.7 26a2 2 0 0 0-2 2c0 1.1.9 2 2 2h4.9c1 0 2-.9 2-2s-1-2-2-2Zm37.8 13.6a2 2 0 0 0-3 0 2 2 0 0 0 0 2.9l3.6 3.5a2 2 0 0 0 2.9 0c.8-.8.8-2.1 0-3ZM10 43.1a2 2 0 0 0 0 2.9c.8.7 2.1.8 3 0l3.4-3.5c.8-.8.8-2.1 0-2.9-.8-.8-2-.8-2.9 0Zm20 3.4c0-1.1-.9-2-2-2a2 2 0 0 0-2 2v4.9c0 1 .9 2 2 2s2-1 2-2Z" />
        </svg>
        
        <svg
          class="absolute left-[4px] top-[4px] z-10 opacity-0 dark:opacity-100"
          viewBox="0 0 24 24"
          fill="none"
          height="16"
          width="16">
          <path
            fill="#000"
            fill-rule="evenodd"
            clip-rule="evenodd"
            d="M8.2 2.2c1-.4 2 .6 1.6 1.5-1 3-.4 6.4 1.8 8.7a8.4 8.4 0 0 0 8.7 1.8c1-.3 2 .5 1.5 1.5v.1a10.3 10.3 0 0 1-9.4 6.2A10.3 10.3 0 0 1 3.2 6.7c1-2 2.9-3.5 4.9-4.4Z" />
        </svg>
      </span>
    </label>
  </div>

  
  <script>
    var darkMode = false;

    
    if (window.matchMedia("(prefers-color-scheme: dark)").matches){darkMode = true}
    

    if (localStorage.getItem("theme") === "dark"){darkMode = true}
    else if (localStorage.getItem("theme") === "light"){darkMode = false}
    if (darkMode){document.documentElement.classList.toggle("dark")}
    var themeSwitch = document.querySelectorAll("[data-theme-switcher]");

    document.addEventListener("DOMContentLoaded", () => {
      [].forEach.call(themeSwitch, function (ts) {
        ts.checked = darkMode ? true : false;
        ts.addEventListener("click", () => {
          document.documentElement.classList.toggle("dark");
          localStorage.setItem(
            "theme",
            document.documentElement.classList.contains("dark") ? "dark" : "light"
          );
        });
      });
    });
  </script>


    </div>

  </nav>
       
</header>

    




  
    <div
      class="search-modal "
      aria-hidden="true"
      style="--color-primary: #121212">
      <div data-target="close-search-modal" class="search-modal-overlay"></div>
      <div
        class="search-wrapper"
        #data-image="true"
        data-description="true"
        data-tags="true"
        data-categories="true">
        <div class="search-wrapper-header">
          <label for="search-modal-input" style="margin-top:-1px">
            <span class="sr-only">search icon</span>
          </label>
        </div>   
        <div class="search-wrapper-body">
          <div class="search-result" data-search-result></div>
          <span class="search-result-empty">
            Type something to search..
          </span>
        </div>
        <div class="search-wrapper-footer">
          <span>
            <kbd>
              <svg
                xmlns="http://www.w3.org/2000/svg"
                width="14"
                height="14"
                fill="currentColor"
                viewBox="0 0 16 16">
                <path
                  d="M3.204 11h9.592L8 5.519 3.204 11zm-.753-.659 4.796-5.48a1 1 0 0 1 1.506 0l4.796 5.48c.566.647.106 1.659-.753 1.659H3.204a1 1 0 0 1-.753-1.659z" />
              </svg>
            </kbd>
            <kbd>
              <svg
                xmlns="http://www.w3.org/2000/svg"
                width="14"
                height="14"
                fill="currentColor"
                style="margin-top:1px;"
                viewBox="0 0 16 16">
                <path
                  d="M3.204 5h9.592L8 10.481 3.204 5zm-.753.659 4.796 5.48a1 1 0 0 0 1.506 0l4.796-5.48c.566-.647.106-1.659-.753-1.659H3.204a1 1 0 0 0-.753 1.659z" />
              </svg>
            </kbd>
            to navigate
          </span>
          <span>
            <kbd>
              <svg
                xmlns="http://www.w3.org/2000/svg"
                width="12"
                height="12"
                fill="currentColor"
                style="display:inline-block;"
                viewBox="0 0 16 16">
                <path
                  fill-rule="evenodd"
                  d="M14.5 1.5a.5.5 0 0 1 .5.5v4.8a2.5 2.5 0 0 1-2.5 2.5H2.707l3.347 3.346a.5.5 0 0 1-.708.708l-4.2-4.2a.5.5 0 0 1 0-.708l4-4a.5.5 0 1 1 .708.708L2.707 8.3H12.5A1.5 1.5 0 0 0 14 6.8V2a.5.5 0 0 1 .5-.5z" />
              </svg>
            </kbd>
            to select
          </span>
          <span class="search-result-info"></span>
          <span data-target="close-search-modal">
            <kbd>ESC</kbd> to close
          </span>
        </div>
      </div>
    </div>
  


  
    <main>
      
  <section class="section-sm" style="padding-top: 15px; padding-bottom: 15px;">
    <div class="container">
      <div class="row justify-center">
        <div class="lg:col-11">
          <div class="content">
            <h2 id="overview">Overview</h2>
<h3 id="analysis-namegossypium-arboreum-a2-sxy1-genome-hau_v1">Analysis Name:Gossypium arboreum (A2) &lsquo;SXY1&rsquo; genome HAU_v1</h3>
<h3 id="methodllumina-and-pacbio">Method:llumina and PacBio</h3>
<h3 id="sourcev1">Source:(v1)</h3>
<h3 id="about-the-assembly">About the assembly:</h3>
<p>In this study, we applied Oxford Nanopore sequencing technology to assemble G. rotundifolium (K2*) &lsquo;K201&rsquo;, G. arboreum (A2) &lsquo;SXY1&rsquo;, and G. raimondii (D5) &lsquo;D502&rsquo; genomes. G. arboreum and G. raimondii genomes have been de novo assembled previously using Illumina and PacBio reads, but both genomes have a number of sequence gaps and require an improvement in assembly contiguity. We generated a total of 304 Gb, 212 Gb, 125 Gb Nanopore sequencing data with a genome coverage 124×, 131×, 167× for K2*, A2 and D5, respectively. We assembled 3,593, 1,173 and 366 contigs for G. rotundifolium, G. arboreum and G. raimondii with a contig length of 2.44 Gb, 1.62 Gb and 0.75 Gb, respectively (Table 1). These initial contigs were polished using Illumina paired-end reads with a genome coverage of 108×, 118×, 132× for K2*, A2 and D5. The contig N50 is 5.33 Mb, 11.69 Mb and 17.04 Mb for K2*, A2 and D5, respectively. The maximum contig has a length of 32.72 Mb, 58.57 Mb and 43.74 Mb. After polishing contig using Illumina reads, we used high-through chromosome conformation capture (Hi-C) data to order and orient contigs, aimed at constructing pseudo chromosomes of each species. In the Hi-C assisted assembly, 2,559, 485 and 201 contigs were placed on the 13 chromosomes of K2*, A2 and D5 genomes, occupying over 99% of genome length.</p>
<p><strong><em>*Should be K12</em></strong></p>
<h3 id="table-1-summary-of-genome-assemblies-and-annotations-of-g-rotundifolium-g-arboreum-and-graimondii">Table 1. Summary of genome assemblies and annotations of G. rotundifolium, G. arboreum and G.raimondii.</h3>
<table>
<thead>
<tr>
<th style="text-align:center">Genomic feature</th>
<th style="text-align:center">G. rotundifolium &lsquo;Grot K201&rsquo;</th>
<th style="text-align:center">G. arboreum &lsquo;Shixiya1&rsquo;</th>
<th style="text-align:center">G. raimondii &lsquo;Grai D502&rsquo;</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center">Total length of contigs, bp</td>
<td style="text-align:center">2,444,364,209</td>
<td style="text-align:center">1,621,008,062</td>
<td style="text-align:center">750,197,587</td>
</tr>
<tr>
<td style="text-align:center">Total length of scaffolds, bp</td>
<td style="text-align:center">2,444,484,509</td>
<td style="text-align:center">1,621,030,562</td>
<td style="text-align:center">750,205,487</td>
</tr>
<tr>
<td style="text-align:center">Total length of gaps, bp</td>
<td style="text-align:center">120,300</td>
<td style="text-align:center">22,500</td>
<td style="text-align:center">7,900</td>
</tr>
<tr>
<td style="text-align:center">Percentage of anchoring</td>
<td style="text-align:center">99.28%</td>
<td style="text-align:center">99.47%</td>
<td style="text-align:center">99.57%</td>
</tr>
<tr>
<td style="text-align:center">Percentage of anchoring and ordering</td>
<td style="text-align:center">93.16%</td>
<td style="text-align:center">98.84%</td>
<td style="text-align:center">99.01%</td>
</tr>
<tr>
<td style="text-align:center">Number of contigs</td>
<td style="text-align:center">3,593</td>
<td style="text-align:center">1,173</td>
<td style="text-align:center">366</td>
</tr>
<tr>
<td style="text-align:center">Number of scaffolds</td>
<td style="text-align:center">2,390</td>
<td style="text-align:center">948</td>
<td style="text-align:center">287</td>
</tr>
<tr>
<td style="text-align:center">Contig N50, bp</td>
<td style="text-align:center">5,326,689</td>
<td style="text-align:center">11,691,474</td>
<td style="text-align:center">17,043,680</td>
</tr>
<tr>
<td style="text-align:center">Contig N90, bp</td>
<td style="text-align:center">621,066</td>
<td style="text-align:center">2,910,421</td>
<td style="text-align:center">3,537,560</td>
</tr>
<tr>
<td style="text-align:center">Scaffold N50, bp</td>
<td style="text-align:center">177,839,665</td>
<td style="text-align:center">129,592,444</td>
<td style="text-align:center">57,716,579</td>
</tr>
<tr>
<td style="text-align:center">Scaffold N90, bp</td>
<td style="text-align:center">115,394,628</td>
<td style="text-align:center">93,157,762</td>
<td style="text-align:center">49,929,625</td>
</tr>
<tr>
<td style="text-align:center">Maximun contig length, bp</td>
<td style="text-align:center">32,728,186</td>
<td style="text-align:center">58,575,076</td>
<td style="text-align:center">43,739,617</td>
</tr>
<tr>
<td style="text-align:center">Maximum scaffold length, bp</td>
<td style="text-align:center">205,722,655</td>
<td style="text-align:center">143,367,608</td>
<td style="text-align:center">63,188,200</td>
</tr>
<tr>
<td style="text-align:center">GC content</td>
<td style="text-align:center">36.38%</td>
<td style="text-align:center">35.16%</td>
<td style="text-align:center">33.23%</td>
</tr>
<tr>
<td style="text-align:center">Percentage of repeat sequences</td>
<td style="text-align:center">80.92%</td>
<td style="text-align:center">68.05%</td>
<td style="text-align:center">57.04%</td>
</tr>
<tr>
<td style="text-align:center">GC content</td>
<td style="text-align:center">36.38%</td>
<td style="text-align:center">35.16%</td>
<td style="text-align:center">33.23%</td>
</tr>
<tr>
<td style="text-align:center">Number of genes</td>
<td style="text-align:center">41,590</td>
<td style="text-align:center">41,778</td>
<td style="text-align:center">40,820</td>
</tr>
</tbody>
</table>
<h3 id="supplementary-table-4-comparing-a2-genome-with-previously-published-genome-version">Supplementary Table 4. Comparing A2 genome with previously published genome version.</h3>
<table>
<thead>
<tr>
<th style="text-align:center">Genomic feature</th>
<th style="text-align:center">HAU_A2</th>
<th style="text-align:center">WHU_A2</th>
<th style="text-align:center">CRI_A2</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center">Total assemblied size, bp</td>
<td style="text-align:center">1,621,008,062</td>
<td style="text-align:center">1,636,985,834</td>
<td style="text-align:center">1,710,104,083</td>
</tr>
<tr>
<td style="text-align:center">Number of total scaffolds</td>
<td style="text-align:center">948</td>
<td style="text-align:center">1,269</td>
<td style="text-align:center">4,516</td>
</tr>
<tr>
<td style="text-align:center">Total length of gaps, bp</td>
<td style="text-align:center">22,500</td>
<td style="text-align:center">116,300</td>
<td style="text-align:center">3,730,000</td>
</tr>
<tr>
<td style="text-align:center">Contig N50, bp</td>
<td style="text-align:center">11,691,474</td>
<td style="text-align:center">1,832,000</td>
<td style="text-align:center">1,100,000</td>
</tr>
<tr>
<td style="text-align:center">Scaffold N50, bp</td>
<td style="text-align:center">129,592,444</td>
<td style="text-align:center">118,841,821</td>
<td style="text-align:center">113,035,596</td>
</tr>
<tr>
<td style="text-align:center">Scaffold N90, bp</td>
<td style="text-align:center">93,157,762</td>
<td style="text-align:center">87,096,684</td>
<td style="text-align:center">162,124</td>
</tr>
<tr>
<td style="text-align:center">Percentage of anchoring and ordering</td>
<td style="text-align:center">98.84%</td>
<td style="text-align:center">92.19%</td>
<td style="text-align:center">85.94%</td>
</tr>
<tr>
<td style="text-align:center">Number of genes</td>
<td style="text-align:center">41,778</td>
<td style="text-align:center">43,278</td>
<td style="text-align:center">40,960</td>
</tr>
</tbody>
</table>
<h2 id="publication">Publication</h2>
<blockquote>
<p><a href="https://academic.oup.com/mbe/article/38/9/3621/6262643?login=false"




 target="_blank"
 


>Wang, M. et al. Comparative genome analyses highlight transposon-mediated genome expansion and the evolutionary architecture of 3D genomic folding in cotton. Mol Biol Evol. 2021 May 11:msab128. doi: 10.1093/molbev/msab128.</a>
</p>
</blockquote>

          </div>
        </div>
      </div>
    </div>
  </section>

    </main>
  
    
    
      <footer class="bg-theme-light dark:bg-darkmode-theme-light">
  <div class="container">
    <div class="row items-center py-0">
      <div class="lg:col-6 mb-8 text-center lg:mb-0">
        <ul>
          
        </ul>
      </div>
      <div class="lg:col-3 mb-8 text-center lg:mb-0 lg:mt-0 lg:text-right">
        <ul class="social-icons">
          
        </ul>
      </div>
    </div>
  </div>
  <div class="border-border dark:border-darkmode-border border-t py-7">
    <div class="text-light dark:text-darkmode-light container text-center">
      <p>
        Designed &amp; Developed by <a href="https://gohugo.io/"




 target="_blank"
 


>Hugo</a>
 and <a href="https://www.netlify.com/"




 target="_blank"
 


>netlify</a>
 | 2024 Jinbiao Cao (曹金彪)
      </p>
    </div>
  </div>
</footer>
      



  
    
      
    
  

  
    
      
    
  

  
    
      
    
  

  
    
      
    
  

  
    
      
    
  

  
    
      
    
  

  
    
      
    
  

  
    
      
    
  

  
    
      
    
  












<script
  crossorigin="anonymous"
  integrity=""
  src="/js/script.js"></script>


<script
  defer
  async
  crossorigin="anonymous"
  integrity=""
  src="/js/script-lazy.js"></script>



<script>
  if ('serviceWorker' in navigator){navigator.serviceWorker.register("/service-worker.js");}
</script>






  








    
  </body>
  
</html>
